siliconcompiler/core.py

```diff
@@ -4090,210 +4053,286 @@ def run(self):
                         for record in self.getkeys('record'):
                             self.unset('record', record, step=step, index=index)
 
-        # Set env variables
-        # Save current environment
-        environment = copy.deepcopy(os.environ)
-
-        for envvar in self.getkeys('option', 'env'):
-            val = self.get('option', 'env', envvar)
-            os.environ[envvar] = val
+    def _load_remote_config(self):
+        '''
+        Load the remote storage config into the status dictionary.
+        '''
+        if self.get('option', 'credentials'):
+            # Use the provided remote credentials file.
+            cfg_file = os.path.abspath(self.get('option', 'credentials'))
```

Comment:
I think all the code related to running the remote flow should go into the client.py and we should just call that and pass in the chip object